admin-api/src/index.js

```diff
@@ -18,6 +18,33 @@ import neo4j from "neo4j-driver";
 import { mergeTypeDefs, mergeResolvers } from '@graphql-tools/merge'
 import { loadFilesSync } from '@graphql-tools/load-files'
 import { runQuery, setDriver, getDriver } from './util/db';
+import packageInfo from "../package.json"
+import winston from "winston";
+import { format } from "winston";
+import DailyRotateFile from "winston-daily-rotate-file";
+
+// Set up the logger
+const logDir = process.env.LOGS_DIR || './../logs'
```

Comment:
Looks good, can we move the definition of the logger out of the index.js to another file?